src/transformers/convert_slow_tokenizer.py

```diff
@@ -1153,7 +1153,13 @@ def tokenizer(self, proto):
         model_type = proto.trainer_spec.model_type
         vocab_scores = self.vocab(proto)
         if model_type == 1:
-            raise RuntimeError("Llama is supposed to be a BPE model!")
+            import tokenizers
+
+            if version.parse(tokenizers.__version__) < version.parse("0.14.0"):
+                tokenizer = Tokenizer(Unigram(vocab_scores, 0))
+            else:
+                tokenizer = Tokenizer(Unigram(vocab_scores, 0, byte_fallback=True))
```

Comment:
Is this safe? The previous check protected against the llama tokenizer being converted when it shouldn't have been

Reply:
The previous check is just too restrictive. It was made because only Llama was there, but for most sentencepiece models now I'd like to re-use the LlamaTokenizer as it's the least buggy. 
If the tokenizer type is Unigram there is not reason to not convert i